cmd/backup-manager/app/util/util.go

```diff
@@ -421,3 +422,21 @@ func ConstructRcloneArgs(conf string, opts []string, command, source, dest strin
 	}
 	return args
 }
+
+// GetContextForSignal get a context for some signals, and the context will become done after any of these signals triggered.
+func GetContextForSignal(op string) (context.Context, context.CancelFunc) {
```

Comment:
this function is used to get a context which will be cancelled by signals defined in this function, i.e. signals that will cause this process to quit.
so how about rename this function to `GetContextForTerminationSignals` or equivalents?

Reply:
done in https://github.com/pingcap/tidb-operator/pull/3696/commits/ec0eeaec9a185024869b758001070d4d468e189f.